### ISSUE TITLE
Bower version isn't up to date

### ISSUE THREAD

**Issue by adarshk7** [insites/cookieconsent#127]
Issue
-------

Installing with `bower install cookieconsent2` installs an older version which references to S3 and that fails.

Temporary solution
--------------------------
I then installed using the latest commit in the repository. That worked as expected as it references to cloudflare.

Solution
-----------

Bower version should be updated to include latest changes.

**Comment by lluiscanals** [comment]
Same happens to me. Can bower be updated? 

Thanks!

**Comment by mattez** [comment]
Still downloads:

`"version": "1.0.9"`
not current
`"version": "1.0.10"`

Please fix it.

**Comment by dennisfrank** [comment]
`version` in `bower.json` is deprecated and should not be used anymore: 

> Use git or svn tags instead. This field is ignored by Bower.

https://github.com/bower/spec/blob/master/json.md#version

**Comment by saurabh-parihar** [comment]
Simple solution for this:

Looking at the source code of this plugin, you can see that at line 293 (Line 291 in older file) it is using a parameter "theme" which is used to download the css file. The older code here refers to the broken Amazon cdn via an internal variable "THEME_BUCKET_PATH". 

If you check the next few lines of the code, luckily, this default css path could be overridden by passing "theme" as a param in the options. Hence, if we use the correct cdn path from the new version of the code, it works as expected.

In short, for people using bower; use this,

`window.cookieconsent_options = {"message":"This website uses cookies to ensure you get the best experience on our website","dismiss":"Got it!","learnMore":"More info","link":null,"theme":"//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/1.0.10/dark-bottom.css"};`

**Comment by Dolondro** [comment]
Hi! Sorry, this project has been left to the side for a while, but we're coming back to it for a few weeks to fix the problems with it.

You should now be able to install this via bower (fingers crossed!) but as part of making the name a bit more sane, you can now get at it using ```bower install cookieconsent``` and installing version 2.0.0.
